_docs-sources/foundations/accounts/enable-control-tower.md

```diff
@@ -0,0 +1,131 @@
+
+:::info
+
+[AWS Control Tower](https://docs.aws.amazon.com/controltower/latest/userguide/what-is-control-tower.html) is an AWS service
+that orchestrates accounts, handling creation, deletion, security policies, and more. It serves as the base layer of
+Gruntwork Account Foundations.
+
+:::
+
+## Prerequisites
+
+In order to enable control tower you will need the resources described in [Prerequisites.](/foundations/accounts/prerequisites)
+
+## Enable AWS Control Tower
+
+:::info
+This Guide should take about 1hr 15min to complete, most of that time will be spent
+waiting on Control Tower Operations at the conclusion of the setup flow.
+:::
+
+### Start Control Tower Setup
+
+1. Sign in to the [AWS management console](https://console.aws.amazon.com) with your administrator user credentials.
+
+1. Navigate to the [AWS Control Tower console](https://console.aws.amazon.com/controltower).
+
+1. Verify that you are working in your desired home Region.
+
+  :::caution
+
+  Your home Region is the AWS Region in which you'll run most of your workloads or store most of your data. It **cannot** be changed after you've set up your AWS Control Tower landing zone. For more information about how to choose a home Region, see [Administrative tips for landing zone setup](https://docs.aws.amazon.com/controltower/latest/userguide/tips-for-admin-setup.html).
+
+  :::
+
+1. Choose **Set up landing zone**.
+
+### Review pricing and select Regions
+
+1. Under **Region deny setting** select Enabled
+   1. This ensures Control Tower policies and controls are unable to be bypassed by using a non-governed region
+
+1. Under **Select additional Regions for governance** Select all regions where you plan to operate.
+
+   ![Region Selections](/img/devops-foundations/account/regions.png)
+
+1. Click **Next** to continue
+
+### Configure Organizational Units (OUs)
+
+1. Rename the "Additional OU" to "Pre-prod"
+
+   ![Configure Organizational Units](/img/devops-foundations/account/configure-ous.png)
+
+1. Click **Next** to continue.
+
+### Configure shared accounts
+
+1. Under **Logs archive account** Enter an email address and rename the `Logs Archive` account to `Logs`
+1. Under **Audit account** Enter an email address and rename the `Audit` account to `Security`
+
+   :::caution
+
+   Account names **cannot** be changed after setting up the landing zone. Ensure the accounts are named appropriately.
+
+   :::
+
+   ![Configure Shared Accounts](/img/devops-foundations/account/log-archive-rename.png)
+
+1. Click **Next** to continue
+
+### Additional configurations
+
+1. Ensure your setting match the screenshot below (These are the defaults)
+   ![Additional Configuration](/img/devops-foundations/account/additional-config.png)
+1. Under **KMS Encryption** Check the box for `Enable and customize encryption settings`
+1. Select the KMS Key you'd like to use to encrypt Control Tower from the dropdown
```

Comment:
Should we call out selecting the key that was made in the prerequisites section (if they did so)?